ocs_ci/ocs/cluster.py

```diff
@@ -1668,6 +1669,47 @@ def check_ceph_health_after_add_capacity(
     ), "Data re-balance failed to complete"
 
 
+def validate_osd_pdb():
+    """
+    Validate creation of PDBs for OSDs.
+    1. Versions lesser than ocs-operator.v4.6.2 have PDBs for each osds
+    2. Versions greater than or equal to ocs-operator.v4.6.2-233.ci have
+    PDBs collectively for osds like rook-ceph-osd
+
+    Returns:
+        bool: True if blocking PDBs are present, false otherwise
+
+    """
+    pdb_obj = ocp.OCP(
+        kind=constants.POD_DISRUPTION_BUDGET, namespace=defaults.ROOK_CLUSTER_NAMESPACE
+    )
+    pdb_obj_get = pdb_obj.get()
+    osd_pdb = []
+    for pdb in pdb_obj_get.get("items"):
+        if (
+            "rook-ceph-mds-ocs-storagecluster-cephfilesystem"
+            not in pdb["metadata"]["name"]
+        ):
+            if "rook-ceph-mon-pdb" not in pdb["metadata"]["name"]:
+                osd_pdb.append(pdb["metadata"]["name"])
+    flag = 0
+    if len(osd_pdb) == 1:
+        logger.info(f"No blocking PDBs created {osd_pdb}")
+    else:
+        logger.info(f"Blocking PDBs are created {osd_pdb}")
+        for osd in range(len(osd_pdb)):
+            allowed_disruptions = (
+                pdb_obj_get.get("items")[osd].get("status").get("disruptionsAllowed")
+            )
+            maximum_unavailable = (
+                pdb_obj_get.get("items")[osd].get("spec").get("maxUnavailable")
+            )
+            if allowed_disruptions and maximum_unavailable != 1:
+                logger.info("Allowed disruptions and maximum_unavailable count matches")
+                flag = 1
+    return bool(flag)
```

Comment:
I think you should use boolean var instead of numeric since you're returning True/False.
In addition, please give this var a meaningful name like "blocking_pdb_exist" or similar